manabi/lock.py

```diff
@@ -4,18 +4,33 @@
 import threading
 import traceback
 import weakref
+from abc import ABC, abstractmethod
 from collections.abc import MutableMapping
 from pathlib import Path
+from typing import Callable
 
 import psycopg2.extensions
 from psycopg2 import InterfaceError, OperationalError, connect
+from psycopg2.extensions import connection as PsycopConnection
```

Comment:
```suggestion
from psycopg2.extensions import connection as PsycopgConnection
```

Renamed type should match the package's name